readthedocs/templates/projects/project_import.html

```diff
@@ -70,7 +70,10 @@
       </span>
 
       <h1>{% trans "Import a Repository" %}</h1>
-
+      {% block non-commercial %}
```

Comment:
The translation block shouldn't be removed. And the block name could be something like `import-tip` or something like that

Reply:
Can I use? `{% blocktrans import-public-repo %}` ?

Reply:
No, those can be nested, take a look at https://github.com/rtfd/readthedocs.org/blob/c0bcb243b14b465695cacf9aab644da23c411ca7/readthedocs/templates/projects/project_import.html#L198-L207